src/data/pages/users.ts

```diff
@@ -0,0 +1,190 @@
+import { sleep } from '../../services/utils'
```

Comment:
Sometimes on page switch - same items are being reshuffled. Let's autogenerate ids or something to prevent that.

Reply:
Still some weirdness.

I would expect:
* You click and loader is shown (but list stays the same).
* Then list updates without shuffling and loader is hidden.

What happens instead:
* You click and loader is shown and list changes.
* Then list updates again **with shuffling** and loader is hidden.

https://github.com/epicmaxco/vuestic-admin/assets/5394573/f23a49c2-94ac-438b-9275-02ace4de397a